Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -439,6 +448,38 @@ void doPublishRawImu() {
 }
 
 /*
+ * Publish bumper data to ros.
+ */
+void doPublishBumpers(void) {
+  ros::Time current_time = ros_nh.now();
+
+  int rawBumperLeft = analogRead(BUMPER_LEFT_PIN);
+  int rawBumperRight = analogRead(BUMPER_RIGHT_PIN);
+
+  double bumperLeft = MAX_FORCE_LEFT_BUMPER 
+                        * ((rawBumperLeft - MIN_RAW_LEFT_BUMPER)
+                            /((double)(MAX_RAW_LEFT_BUMPER - MIN_RAW_LEFT_BUMPER)));
+  double bumperRight = MAX_FORCE_RIGHT_BUMPER 
+                        * ((rawBumperRight - MIN_RAW_RIGHT_BUMPER)
+                            /((double)(MAX_RAW_RIGHT_BUMPER - MIN_RAW_RIGHT_BUMPER)));
+
+  ros_bumper_left_msg.header.stamp = current_time;
+  // TODO(mwomack): set header frame to what?
```

Comment:
Set the frames to /bumper_left and /bumper_right.
We'll have to define the transformation in the urdf.